### PR TITLE
New package: python-hyper-h2-3.0.1

### PR DESCRIPTION
**Pull request by ndowens**
Adding python--hyperh2 and python-hyperframe  to PR, as they depends on python-hpack

### CODE REVIEW

File: srcpkgs/python-hpack/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+python3-hpack_package() {
+	noarch=yes
+	pycompile_module="hpack"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove "usr/lib/python3*"
+	}
+}
+
+post_install() {
```
Comment (maxice8) [inline srcpkgs/python-hpack/template]: build steps come before subpackage declarations

Comment (ndowens) [reply srcpkgs/python-hpack/template]: Fixed

**Comment by maxice8** [comment]
separate commits for the packages

**Comment by ndowens** [comment]
> separate commits for the packages

K thought should have been done together since one depends on the other

**Comment by maxice8** [comment]
On Thu, Jan 10, 2019 at 01:15:58PM -0800, Nathan wrote:
> > separate commits for the packages
> 
> K thought should have been done together since one depends on the other
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/6846#issuecomment-453256634

Dependencies are a reason for keeping them in the same PR but each package should have its own commit

**Comment by ndowens** [comment]
Done

**Pull request merged**